cohorts/plot.py

```diff
@@ -43,7 +44,18 @@ def stripboxplot(x, y, data, ax=None, **kwargs):
         **kwargs
     )
 
-def fishers_exact_plot(data, condition1, condition2, ax=None):
+def sided_str_from_alternative(alternative):
+    if alternative is None:
+        raise ValueError("Must pick an alternative")
+    if alternative == "two-sided":
+        return alternative
+    # alternative hypothesis: condition is 'less' or 'greater' than no-condition
+    op_str = ">" if alternative == "greater" else "<"
+    return "one-sided: %s %s not %s" % (condition, op_str, condition)
```

Comment:
Yikes, that's confusing; especially since it was working as far as I could tell.